src/main/java/com/google/devtools/build/lib/analysis/FileProvider.java

```diff
@@ -53,6 +55,13 @@ public boolean isImmutable() {
     return Depset.of(Artifact.class, filesToBuild);
   }
 
+  @Override
+  public void debugPrint(Printer printer, StarlarkThread thread) {
+    printer.append("file_provider(files_to_build = ");
```

Comment:
For consistency with FilesToRunProvider, this should probably be "FileProvider".

I'd prefer more Starlark names ending with Info, but they aren't used at the moment anywhere.